src/Psalm/Checker/ProjectChecker.php

```diff
@@ -181,7 +182,7 @@ public function __construct(
             $debug_output
         );
 
-        if (!in_array($output_format, [self::TYPE_CONSOLE, self::TYPE_JSON, self::TYPE_EMACS, self::TYPE_XML], true)) {
+        if (!in_array($output_format, [self::TYPE_CONSOLE, self::TYPE_PYLINT, self::TYPE_JSON, self::TYPE_EMACS, self::TYPE_XML], true)) {
```

Comment:
This is a too-long line. Why not make a constant array called `SUPPORTED_OUTPUT_TYPES` with this list?